README.md

```diff
@@ -86,6 +88,37 @@ The merge_core_pattern function is designed to take a patterns DF and cross exam
 The merge_socialDist_by_dates function is designed to merge the social distancing data from a given start_date to a given end_date. The resulting pandas DF will be a DF of all social distancing data from the start_date to the end_date. All Pandas arguments and keywords arguments can be passed into this function.
 * start_date and end_date are strings formated as: "year-month-day"
 
+## cbg_functions
+
+## Usage
+
+```python
+from safegraph_py_functions import cbg_functions as sgpy
+
+sgpy.test_me_cbg() # returns 'Hello World' to ensure you have downloaded the library
+sgpy.help_cbg() # returns a list of all active functions and their arguments in the cbg_functions library
+sgpy.get_cbg_field_descriptions(year) - This function creates a reference table of Census data columns and their definitions
+# etc. . . 
+```
+
+## Functions
+
+#### get_drive_id(year, drive_ids)
+
+This function is used to pull input files from Google Drive. It requires input of a year (of 2016, 2017, 2018, or 2019) and a dictionary of Google Drive IDs with the requisite respective data. This function is used automatically within other functions, so knowledge of said dictionary is not necessary.
+
+#### pd_read_csv_drive(id, drive, dtype = None)
+
+This function is used to pull input files from Google Drive into Pandas DataFrames. This function takes the output of the chosen year from the previous function, get_drive_id, as its first required input. The second required input is a Google Drive object, automatically created within the functions that use these functions.
+
+#### get_cbg_field_descriptions(year=2019)
```

Comment:
Add an example usage and the first five rows or so of output under these two functions.